jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/AssetsCountManager.java

```diff
@@ -39,7 +39,17 @@ public List<Map<String, String>> populateAssetCount(){
             String assetCount;
             try {
                 if(platform.equals("azure")) {
-                    assetCount = AssetGroupUtil.fetchAssetCount(ASSET_API_URL, token, platform, getSubscriptionsForTenant(accountId));
+                    String combinedSubscriptionStr = getSubscriptionsForTenant(accountId);
+                    String[] subscriptionArray = combinedSubscriptionStr.split(",");
+                    Integer totalAssetCount = 0;
+                    for(String subscriptionStr : subscriptionArray){
+                        String assetCountForSubscription = AssetGroupUtil.fetchViolationsCount(ASSET_API_URL, token, platform, subscriptionStr);
```

Comment:
this should be fetchAssetCount instead of violationCount

Reply:
updated it to fetachAssetCount